app/src/main/java/id_iot/demo/CategoryFragment.java

```diff
@@ -0,0 +1,32 @@
+package id_iot.demo;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+public class CategoryFragment extends Fragment {
+
+    RecyclerView recyclerView;
+    RecyclerView.LayoutManager layoutManager;
```

Comment:
do you actually need to keep this as a field of class. it seems you don't even need this as variable. You can simply use recyleView.setLayoutManager(new LinearLayoutManager(getContext))